go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/libp2p/go-libp2p-core v0.3.0
 	github.com/libp2p/go-libp2p-discovery v0.2.0
 	github.com/libp2p/go-libp2p-kad-dht v0.5.0
+	github.com/libp2p/go-libp2p-peer v0.2.0
```

Comment:
This should not be here. Is some other file importing this package?

Reply:
Removed.